admin_guide/overcommit.adoc

```diff
@@ -11,64 +11,175 @@ toc::[]
+distributed among the containers in a 2:1 ratio. If a container specified a
+limit, it will be throttled to not use more CPU than the specified limit.
 
-For each compute resource, a container is divided into one of 3 QoS classes with decreasing order of priority:
+CPU requests are enforced using the CFS shares support in the Linux kernel. By
+default, CPU limits are enforced using the CFS quota support in the Linux kernel
+over a 100ms measuring interval, though link:#enforcing-cpu-limits[this can be
+disabled].
 
-1. Guaranteed
-2. Burstable
-3. BestEffort
+[[overcommit-memory]]
+=== Memory
 
-If limits and optionally requests are set (not equal to 0) for all resources and they are equal, then the container is classified as Guaranteed.
+A container is guaranteed the amount of memory it requests. A container may use
+more memory than requested, but once it exceeds its requested amount, it could
+be killed in a low memory situation on the node.
 
-If requests and optionally limits are set (not equal to 0) for all resources, and they are not equal, then the container is classified as Burstable.
+If a container uses less memory than requested, it will not be killed unless
+system tasks or daemons need more memory than was accounted for in the node's
+resource reservation. If a container specifies a limit on memory, it is
+immediately killed if it exceeds the limited amount.
 
-If requests and limits are not set for any of the resources, then the container is classified as BestEffort.
+[[qos-classes]]
+== Quality of Service Classes
 
-Memory is an incompressible resource, so in low-memory situations, containers are killed that have the lowest priority.
+A node is _overcommitted_ when it has a pod scheduled that makes no request, or
+when the sum of limits across all pods on that node exceeds available machine
+capacity.
+
+In an overcommitted environment, it is possible that the pods on the node will
+attempt to use more compute resource than is available at any given point in
+time. When this occurs, the node must give priority to one pod over another. The
+facility used to make this decision is referred to as a Quality of Service (QoS)
+Class.
+
+For each compute resource, a container is divided into one of three QoS classes
+with decreasing order of priority:
+
+.Quality of Service Classes
+[options="header",cols="1,1,5"]
+|===
+|Priority |Class Name |Description
+
+|1 (highest)
+|*Guaranteed*
+|If limits and optionally requests are set (not equal to 0) for all resources
+and they are equal, then the container is classified as *Guaranteed*.
+
+|2
+|*Burstable*
+|If requests and optionally limits are set (not equal to 0) for all resources,
+and they are not equal, then the container is classified as *Burstable*.
+
+|3 (lowest)
+|*BestEffort*
+|If requests and limits are not set for any of the resources, then the container
+is classified as *BestEffort*.
+|===
+
+Memory is an incompressible resource, so in low memory situations, containers
```

Comment:
I like this, but maybe order the bullets below in the same order as the table above?